docs/guide/siteaccess.md

```diff
@@ -55,6 +55,20 @@ Collects SiteAccesses into groups that can be used later for configuration.
 
 The rule or set of rules by which SiteAccesses are matched. See [SiteAccess matching](#siteaccess-matching) for more information.
 
+!!! enterprise
+
+    ### SiteAccess selection in Page Builder
+
+    To define which SiteAccesses are available in the submenu in Page Builder, use the following configuration:
```

Comment:
`use the following convention, where siteaccess_list means an array of siteaccess identifiers`